scene/2d/tile_map.h

```diff
@@ -225,6 +195,7 @@ class TileMap : public Node2D {
 
 	void _set_old_cell_size(int p_size) { set_cell_size(Size2(p_size, p_size)); }
 	int _get_old_cell_size() const { return cell_size.x; }
+	Vector2i _vec2i_to_quadrant(const Vector2i &v, const int &p_quadrant_size) const;
```

Comment:
`_vec2i_to_quadrant` isn't clear, since it takes a Vector2i and returns a Vector2i.
It should be `_pos_to_quadrant` or similar.

Reply:
it could be `_map_to_quadrant` IMO.

Reply:
Sounds good.